pom.xml

```diff
@@ -174,7 +174,7 @@
         <dep.kafka-clients.version>3.3.2</dep.kafka-clients.version>
         <dep.casandra.version>4.17.0</dep.casandra.version>
         <dep.minio.version>8.5.6</dep.minio.version>
-        <dep.iceberg.version>1.3.1</dep.iceberg.version>
+        <dep.iceberg.version>1.4.0</dep.iceberg.version>
```

Comment:
Could you remove TrinoParquetUtil class? We introduced the class temporarily as workaround of https://github.com/apache/iceberg/pull/8559.

Reply:
Thanks for reminding me, and certainly, less is more.